memgpt/cli/cli_config.py

```diff
@@ -475,9 +477,17 @@ def configure():
     config.save()
 
 
+class ListChoice(str, Enum):
+    agents = "agents"
+    humans = "humans"
+    personas = "personas"
+    sources = "sources"
+
+
 @app.command()
-def list(option: str):
-    if option == "agents":
+def list(arg: Annotated[ListChoice, typer.Argument]):
+    typer.echo(f"You chose {arg}")
```

Comment:
Do you have a screenshot for what this looks like in-context?

Wondering if this should be color coded to match other `cli.py` / `configure` elements.

Reply:
Whoops that typer.echo statement was put there while debugging. I think we should remove it. But I will look into coloring and I will provide a screenshot later this morning.